docs/workflows.md

```diff
@@ -43,11 +45,37 @@ syntax --
 TYPE: VAR_NAME = FILE_NAME
 ```
 
--- where `TYPE` is the type of the file ("text", "json", "arraybuffer", or "blob"), `VARNAME`
-is the variable into which you want to load the data, and `FILE_NAME` is the
-name of the file you uploaded.
-
-This notebook [shows how to load and use an image blob](https://alpha.iodide.io/notebooks/127/).
+-- where `TYPE` is the type of the file ("text", "json", "arraybuffer", or
+"blob"), `VARNAME` is the variable into which you want to load the data, and
+`FILE_NAME` is the name of the file you uploaded.
+
+This notebook [shows how to load and use an image
+blob](https://alpha.iodide.io/notebooks/127/).
+
+### scheduling file uploads
```

Comment:
On request, here is some documentation. Feel free to insert suggestions rather than comment on this to save time.